### ISSUE TITLE
Column data do not wrap as given width

### ISSUE THREAD

**Issue by NaingMyoThawNMYT** [andrey-ushakov/esc_pos_bluetooth#77]
Column data do not wrap as given width.
![4zi92](https://user-images.githubusercontent.com/46706152/125198129-a5cb6f00-e286-11eb-824e-d87cebc4e8e1.jpg)

**Comment by ozhankucuk** [comment]
Yes, I have same issue. I think it is about esc pos command.If there ise any solution.
Thanks
